README.md

```diff
@@ -1762,16 +1762,22 @@ yarn install
 
 ### Build and Test Commands
 
-| Command               | Description                    |
-| --------------------- | ------------------------------ |
-| `yarn build`          | Build the SDK with a sourcemap |
-| `yarn test`           | Run unit tests     |
-| `yarn lint`           | Run eslint linting             |
-| `yarn start`          | Start internal test app        |
+| Command               | Description                     |
+| --------------------- | ------------------------------- |
+| `yarn build`          | Build the SDK with a sourcemap  |
+| `yarn start`          | Start internal test app         |
+| `yarn lint`           | Run eslint linting              |
+| `yarn test:unit`      | Run only unit tests             |
+| `yarn test:e2e`       | Run only E2E (end-to-end) tests |
+| `yarn test`           | Run all tests                   |
+
+#### Test Environment
+
+Before running the E2E tests, you will need to setup a test environment. See [test/e2e/README](test/e2e/README.md) for more information.
 
 #### Test App
 
-Implements a simple SPA application to demonstrate functionality and provide for manual testing. [See here for more information](test/app/README.md).
+The internal test app can be run manually using `yarn start`. This is useful for troubleshooting and manual testing. See [test/app/README](test/app/README.md) for more information on the test app.
```

Comment:
Hmm...IIRC we're doing some lightly funky bootstrapping to allow for the configs to change on the fly, which means this is a BAD example app for users to use as the basis for their code.  

If (_if_) I'm correct, we should probably mention "Because this test app is set up to dynamically change configuration, users should not use this test app as the basis for their applications.  Instead, use the example usage outlined elsewhere in this README."